### PR TITLE
Kicad 5.1.0

### PR DESCRIPTION
**Pull request by zpuskas**

### CODE REVIEW

File: sci-electronics/kicad-footprints/kicad-footprints-5.1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (Zlogene) [inline sci-electronics/kicad-footprints/kicad-footprints-5.1.0.ebuild]: why not 7?

File: sci-electronics/kicad-i18n/kicad-i18n-5.1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
```
Comment (Zlogene) [inline sci-electronics/kicad-i18n/kicad-i18n-5.1.0.ebuild]: why not 7?

File: sci-electronics/kicad-i18n/kicad-i18n-5.1.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+DESCRIPTION="Electronic Schematic and PCB design tools GUI translations."
```
Comment (Zlogene) [inline sci-electronics/kicad-i18n/kicad-i18n-5.1.0.ebuild]: no dot at the end

File: sci-electronics/kicad-i18n/kicad-i18n-5.1.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+HOMEPAGE="https://github.com/KiCad/kicad-i18n"
+SRC_URI="https://github.com/KiCad/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="CC-BY-SA-4.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE=""
+
+DEPEND="sys-devel/gettext"
```
Comment (Zlogene) [inline sci-electronics/kicad-i18n/kicad-i18n-5.1.0.ebuild]: with EAPI=7 thi si BDEPEND

File: sci-electronics/kicad-packages3d/kicad-packages3d-5.1.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (Zlogene) [inline sci-electronics/kicad-packages3d/kicad-packages3d-5.1.0.ebuild]: why not 7?

File: sci-electronics/kicad-templates/kicad-templates-5.1.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (Zlogene) [inline sci-electronics/kicad-templates/kicad-templates-5.1.0.ebuild]: and again

File: app-doc/kicad-doc/kicad-doc-5.1.0.ebuild [hunk 7]
```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (Zlogene) [inline app-doc/kicad-doc/kicad-doc-5.1.0.ebuild]: and again, 7

File: app-doc/kicad-doc/kicad-doc-5.1.0.ebuild [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+LANG_USE=" l10n_ca l10n_de l10n_en l10n_es l10n_fr l10n_id l10n_it l10n_ja l10n_pl l10n_ru l10n_zh"
+IUSE+=${LANG_USE}
+REQUIRED_USE="|| ( html pdf ) ^^ ( ${LANG_USE} )"
+unset LANG_USE
+
+DEPEND="
+	>=app-text/asciidoc-8.6.9
+	>=app-text/dblatex-0.3.10
+	>=app-text/po4a-0.45
+	>=sys-devel/gettext-0.18
```
Comment (Zlogene) [inline app-doc/kicad-doc/kicad-doc-5.1.0.ebuild]: and this is BDEPEND

File: sci-electronics/kicad-i18n/kicad-i18n-5.1.0.ebuild (hunk 3 above)
Comment (pager72) [reply sci-electronics/kicad-i18n/kicad-i18n-5.1.0.ebuild]: This stands in violation of EAPI?

File: sci-electronics/kicad/kicad-5.1.0.ebuild [hunk 9]
```diff
@@ -0,0 +1,127 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
```
Comment (hsoft) [inline sci-electronics/kicad/kicad-5.1.0.ebuild]: Doesn't the 5.1.0 announcement mention python 3 support?

Comment (hsoft) [reply sci-electronics/kicad/kicad-5.1.0.ebuild]: Oh, wxpython, nevermind. We should update that...

**Comment by zpuskas** [comment]
Updated EAPI everywhere it was possible.

**Pull request closed without merging**